stripe/resource_stripe_meter.go

```diff
@@ -0,0 +1,299 @@
+package stripe
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/stripe/stripe-go/v78"
+	"github.com/stripe/stripe-go/v78/client"
+)
+
+func resourceStripeMeter() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   resourceStripeMeterRead,
+		CreateContext: resourceStripeMeterCreate,
+		UpdateContext: resourceStripeMeterUpdate,
+		DeleteContext: resourceStripeMeterDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"default_aggregation": {
```

Comment:
Could you please include id like it is in other resources?

Reply:
done